dcpy/builds/metadata.py

```diff
@@ -39,6 +39,12 @@ def build_name(name: str | None = None) -> str:
     return name.lower().replace("-", "_")
 
 
+def build_tests_name(build_name: str) -> str:
+    if os.environ.get("BUILD_ENGINE_SCHEMA"):
+        return os.environ["BUILD_ENGINE_SCHEMA_TESTS"]
```

Comment:
Is there a case where we have `BUILD_ENGINE_SCHEMA` but not `BUILD_ENGINE_SCHEMA_TESTS` env var? If so, we should probably check for both env variable in the if statement or handle the exception?

Reply:
oops that's a typo! both lines should be `BUILD_ENGINE_SCHEMA_TESTS`